box_it_script/boxIt.js

```diff
@@ -0,0 +1,72 @@
+function topBorder(topWidth) {
```

Comment:
Functions are actions. I suggest you name functions starting with a verb. e.g. `function drawTopBorder ...`